client/transport/src/ws/mod.rs

```diff
@@ -178,7 +178,7 @@ pub enum WsHandshakeError {
 #[derive(Debug, Error)]
 pub enum WsError {
 	/// Error in the WebSocket connection.
-	#[error("WebSocket connection error: {}", 0)]
+	#[error("WebSocket connection error: {0}")]
```

Comment:
the old code will print `0` and not the actual error :shushing_face:

Reply:
oops!